ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -35,26 +35,46 @@ def toProtocolElement(self):
 
 
 class SimulatedPhenotypeAssociationSet(AbstractPhenotypeAssociationSet):
-    def __init__(self, parentContainer, localId, randomSeed):
+    def __init__(self, parentContainer, localId, randomSeed,
+                 numAssociations=2):
```

Comment:
@dcolligan thanks as usual for your review.  I'm assuming I would follow the handoff between frontend and dataRepository found [here](https://github.com/ga4gh/server/blob/g2p/ga4gh/frontend.py#L235-L236)